lib/kafka-consumer.js

```diff
@@ -369,7 +379,7 @@ KafkaConsumer.prototype.unsubscribe = function() {
  * is fetched.
  */
 KafkaConsumer.prototype.consume = function(number, cb) {
-  var timeoutMs = this._consumeTimeout || 1000;
+  var timeoutMs = this._consumeTimeout || DEFAULT_CONSUME_LOOP_TIMEOUT_DELAY;
```

Comment:
Here you should use `DEFAULT_CONSUME_TIME_OUT` instead of `DEFAULT_CONSUME_LOOP_TIMEOUT_DELAY`